include/effects/Saturation.h

```diff
@@ -65,14 +65,17 @@ namespace openshot
 
 	public:
 		Keyframe saturation;	///< The color saturation: 0.0 = black and white, 1.0 = normal, 2.0 = double saturation
```

Comment:
I would change this description, as it's no longer really "**The**" saturation, it's _a_ saturation. Maybe something like "Overall color saturation" or "Common color saturation (all channels)"?

Reply:
(Also, "black and white" is terrible and should probably be "grayscale". Though, again, not your fault.)

Reply:
> I would change this description, as it's no longer really "**The**" saturation, it's _a_ saturation. Maybe something like "Overall color saturation" or "Common color saturation (all channels)"?

Fixed by ffab5705543b3739e1f4b8bcdef0460eb8947952.

Reply:
> (Also, "black and white" is terrible and should probably be "grayscale". Though, again, not your fault.)

Fixed by fb932ae09c1b76f561966e24f2eba94481907504.